docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -487,31 +439,35 @@ addressed in due course.
    etcd](https://coreos.com/etcd/docs/latest/admin_guide.html). The etcd data
    directory configured by kubeadm is at `/var/lib/etcd` on the master.
 
-1. The `HostPort` and `HostIP` functionality does not work with kubeadm due to
-   that CNI networking is used, see issue
-   [#31307](https://github.com/kubernetes/kubernetes/issues/31307).
 
-   Workaround: use the [NodePort feature of
-   services](/docs/concepts/services-networking/service/#type-nodeport) instead, or use
-   HostNetwork.
 
-1. Some users on RHEL/CentOS 7 have reported issues with traffic being routed
-   incorrectly due to iptables being bypassed. You should ensure
-   `net.bridge.bridge-nf-call-iptables` is set to 1 in your sysctl config, eg.
+## Troubleshooting {#troubleshooting}
 
-   ``` bash
-   cat /etc/sysctl.d/k8s.conf
-   ```
+You may have trouble in the configuration if you see Pod statuses like `RunContainerError`,
+`CrashLoopBackOff` or `Error`.
 
-   Should have:
+1. **There are Pods in the `RunContainerError`, `CrashLoopBackOff` or `Error` state**
+    Right after `kubeadm init` there should not be any such Pods. If there are Pods in
+    such a state _right after_ `kubeadm init`, please open an issue in the kubeadm repo.
+    `kube-dns` should be in the `Pending` state until you have deployed the network solution.
+    However, if you see Pods in the `RunContainerError`, `CrashLoopBackOff` or `Error` state
+    after deploying the network solution and nothing happens to `kube-dns`, it's very
+    likely that the Pod Network solution that you installed is somehow broken. You
+    might have to grant it more RBAC privileges or use a newer version. Please file
+    an issue in the Pod Network providers' issue tracker and get the issue triaged there.
 
-   ```
-   net.bridge.bridge-nf-call-ip6tables = 1
-   net.bridge.bridge-nf-call-iptables = 1
-   ```
+1. **The `kube-dns` Pod is stuck in the `Pending` state forever**
+    This is expected and part of the design. kubeadm is network provider-agnostic, so you should
+    install the pod network solution of choice  is normal, you have to install a Pod Network
```

Comment:
add a link to the network addons page so that users here can act on this suggestion